tvos/plugin/android/src/main/kotlin/com/tva/quickbrickplayerplugin/TVAQuickBrickPlayerView.kt

```diff
@@ -409,6 +409,12 @@ class TVAQuickBrickPlayerView(context: Context, attrs: AttributeSet?) : FrameLay
 
     }
 
+    private fun getCustomUserId(): String? {
+        var id = LocalStorage.storageRepository?.get(CUSTOM_USER_ID_KEY, CUSTOM_USER_ID_NAMESPACE) ?: ""
+        return id.replace("\"", "")
```

Comment:
This is good, but add a comment why we need to do it.

Reply:
On JS side when you persist a value it’s stringified to make it safe for storing.
JSON.stringify("String") // "\"String\""